lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -233,6 +237,10 @@ filePermissionIsSafe s umask = not fileWorldWritable && fileGroupWritable <= uma
     fileGroupWritable = fileMode s .&. 0o020 == 0o020
     umaskGroupWritable = umask .&. 0o020 == 0
 
+inProjectProc :: MonadObelisk m => String -> [Bash] -> m ()
+inProjectProc shellName command = withProjectRoot "." $ \root ->
+  projectProc root True shellName (Just command)
```

Comment:
Why do we have this function? Why do we have `inProjectShell`? All it adds is `withProjectRoot "."`? Seems like we should either never use this or always use it... Please investigate a bit and let me know what you think. I'd rather not perpetuate the pattern if it's not a good one.

Reply:
Right, so the reason we can't always pass around  `[Bash]`, is because sometimes we want to use actual unescaped bash constructs, eg. [this haddock script](https://github.com/obsidiansystems/obelisk/blob/acc5e6db09462d76c7e7eaff3aa3634a7ce0c371/lib/command/src/Obelisk/Command.hs#L397), which wouldn't make sense if it were escaped.

That's why I added a separate typesafe version for simple programs.

Reply:
Perhaps we can add the intended use as a comment on that function? Something related to `separate typesafe version for simple programs`?

Reply:
I deleted `inProjectShell` in latest `qa` because it wasn't used in a uniform way. That answers this question in my mind. So we should delete this counterpart here.